open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-api/pom.xml

```diff
@@ -49,11 +49,11 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
-        <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-            <scope>test</scope>
-        </dependency>
+		<dependency>
```

Comment:
you'll probably need to mirror this change in the build.gradle. N

Reply:
It is just formatting, testng was not removed, it is still in gradle dependency: 
testImplementation 'org.testng:testng:7.1.0'

But I've noticed gradle and maven are using different versions of testng.
      <dependency>
        <groupId>org.testng</groupId>
        <artifactId>testng</artifactId>
        <version>7.3.0</version>
        <scope>test</scope>
      </dependency>
Should I define 7.3.0 for gradle?

Reply:
Hi, After a merge to master on Friday, the gradle versions are now all consolidated and can be found in the top level build.gradle (search for 'constraints'). That is now version 7.3

The versions specified in individual files have been removed -- it was never the right approach, but the first step of the migration (maven->gradle tool created them that way )

Reply:
Then it should be good.